core/vm/contract.go

```diff
@@ -105,20 +110,6 @@ func (c *Contract) validJumpSubdest(udest uint64) bool {
 // isCode returns true if the provided PC location is an actual opcode, as
 // opposed to a data-segment following a PUSHN operation.
 func (c *Contract) isCode(udest uint64) bool {
-	// Do we have a contract hash already?
```

Comment:
I think  you were also bitten by the regression fixed here: https://github.com/ethereum/go-ethereum/pull/21411 . That fix might actually have had a similar effect as this whole PR...?

Reply:
yeah, I think we were, I remember looking at that code...